websites/manopozicija.lt/deploy.yml

```diff
@@ -124,8 +127,8 @@
   - name: install letsencrypt
     pip: name={{ item }} virtualenv=/opt/letsencrypt state=latest
     with_items:
-    - letsencrypt
-    - letsencrypt-apache
+    - certbot
+    - certbot-apache
```

Comment:
Probably, we can use cert bot from apt?

Reply:
Yes, but.  The certs are already there, an /opt/letsencrypt already exists, it has certbot and certbot-apache installed.  All I want is for this playbook to not touch anything related to certs/certbot.

I didn't want to remove the apt/certbot bits from the playbook in case you later decide to move to a server not managed by me, since they may be convenient for you then.

I want the same setup on all my servers for convenience.  Distro-provided certbot was too outdated when I initially set things up.  Later on some changes like the sudden deprecation of tls-sni-01 that required me to quickly upgrade certbot-apache everywhere seemed to justify the decision not to rely on distro packages either.  I think nowadays there's a ppa but migrating everything is work I don't want to do right now, for uncertain gains.